tests/com.b2international.snowowl.test.commons/src/com/b2international/snowowl/test/commons/SnomedContentRule.java

```diff
@@ -77,4 +81,34 @@ private void checkBranch() {
 		}
 	}
 
+	private void createCodeSystemIfNotExist() {
+		if (!SnomedTerminologyComponentConstants.SNOMED_SHORT_NAME.equals(codeSystem.getShortName())) {
```

Comment:
I think we can remove this part from the tests. The boot process should create the primary code system and the tests would only have to create the extensions, if they require.